ChangeLog

```diff
@@ -1,3 +1,35 @@
+# 2024-10-14 Version 3.9.0
+
+We're proud to present the newest release of FreeRDP.
+This one brings some major code cleanup (we've addressed lots of clang-tidy
+warnings) as well as some highly anticipated new features.
+We also did update the API documentation quite a bit (still incomplete though,
+help always welcome ;))
+
+So, what is new:
+* Support for RDPEAR /remoteGuard option, finally kerberos authentication
+  without passwords (at least in restricted admin mode)
```

Comment:
I'm not comfortable with this description of RCG, as it's more that with RCG the server does not have the credentials. And with the changes done for RCG, I think we can logon with restricted admin from the ccache.

Reply:
@hardening ok, will rephrase that.

Reply:
ok, updated, hope it sounds better now :grimacing: